builtin/types/coin/coin.proto

```diff
@@ -24,6 +25,14 @@ message InitialAccount {
 
 message InitRequest {
     repeated InitialAccount accounts = 1;
+    Policy policy = 2;
+}
+
+message Policy {
+    uint64 deflation_factor_numerator = 1 [(gogoproto.customname)="DeflationFactorNumerator"];
```

Comment:
There's no need for these `customname` tags, the code generator will name the Go fields like this anyway.

Reply:
Done.